examples/campaign_management/update_campaign_criterion_bid_modifier.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Sets a bid modifier for the mobile platform on the given campaign."""
+
+import argparse
+import sys
+
+import google.ads.google_ads.client
+from google.api_core import protobuf_helpers
+
+
+_BID_MODIFIER = 1.5
+
+
+def main(client, customer_id, criterion_id):
+    campaign_criterion_service = client.get_service(
+        'CampaignCriterionService', version='v2')
+
+    criterion_rname = campaign_criterion_service.campaign_criteria_path(
+        customer_id, criterion_id)
```

Comment:
I'm surprised this worked without the campaign id.

Reply:
The campaign_id is already part of the criterion_id, ie. 

Campaign resource name: customers/1234567890/campaigns/**2345678901**
Campaign Criteria resource name: customers/1234567890/campaignCriteria/**2345678901**~1003.